Sources/AppHost/OsmAndAppImpl.mm

```diff
@@ -430,8 +432,8 @@ - (BOOL) initializeImpl
     OALog(@"Weather Forecast path: %@", _weatherForecastPath);
 
     // Unpack app data
+    _appSettingsLoadedObservable = [[OAObservable alloc] init];
     _data = [NSKeyedUnarchiver unarchiveObjectWithData:[[NSUserDefaults standardUserDefaults] dataForKey:kAppData]];
```

Comment:
'unarchiveObjectWithData:' is deprecated: first deprecated in iOS 12.0 - Use +unarchivedObjectOfClass:fromData:error: instead

Need to remove the deprecated code. Check and replace with implementation:

```
    NSError *error;
    NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc]
                                     initForReadingFromData:[[NSUserDefaults standardUserDefaults] dataForKey:kAppData]
                                     error:&error];
    [unarchiver setRequiresSecureCoding:NO];
    
    _data = [unarchiver decodeObjectForKey:NSKeyedArchiveRootObjectKey];
    if (error)
    {
        NSLog(@"error unarchiving data: %@", error.localizedDescription);
    }
```